dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/gdv/GdvFramework.kt

```diff
@@ -111,15 +111,14 @@ class GdvFramework : InDevelopmentPavedRoadFramework(
     override fun customizeHighLevelIntermediateRepresentation(framework: Framework) {
         setGroupsThatAreExpandedOnPageLoad(framework)
         overwriteFakeFixtureGenerationForDataDate(framework)
-        val berichtspflichtUndEinwilligungZurVeroeffentlichung = framework.root
+        val berichtsPflicht = framework.root
```

Comment:
If this was not the breaking point. I would tend to use the longer correcter name here, although it is likely that we will return to this field name in the long run.